src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -158,6 +160,45 @@ export function prefillTransformer(pages, formData, metadata) {
   };
 }
 
+/**
+ * Transforms the related disabilities object into an array of strings
+ * @param {Object} object - The object with dynamically generated property names
+ * @return {Array} - An array of the property names with truthy values
+ */
+export function transformRelatedDisabilities(object) {
+  return Object.keys(object).filter(key => object[key]);
+}
+
+export function transform(formConfig, form) {
+  // Have to do this first or it messes up the results from transformRelatedDisabilities for some reason.
```

Comment:
hmm, do you know why?

Reply:
Why? No :disappointed:. It was changing `relatedDisabilities: ['Stress Disorder']` to
```
"relatedDisabilities":[{"0":"S","1":"t","2":"r","3":"e","4":"s","5":"s","6":" ","7":"D","8":"i","9":"s","10":"o","11":"r","12":"d","13":"e","14":"r"}]}]
```